src/test/addrman_tests.cpp

```diff
@@ -268,32 +268,32 @@ BOOST_AUTO_TEST_CASE(addrman_new_collisions)
 
 BOOST_AUTO_TEST_CASE(addrman_tried_collisions)
 {
-    AddrManTest addrman;
+    auto addrman = std::make_unique<AddrMan>(std::vector<bool>(), /* makeDeterministic */ true, /* consistency_check_ratio */ 100);
```

Comment:
clang-tidy expects the comments to be formatted as:

```suggestion
    auto addrman = std::make_unique<AddrMan>(std::vector<bool>(), /*deterministic=*/true, /*consistency_check_ratio=*/100);
```